src/Module.php

```diff
@@ -25,6 +24,11 @@ class Module extends \Dns_Module implements ProviderInterface {
 		 * apex markers are marked with @
 		 */
 		protected const HAS_ORIGIN_MARKER = true;
+        protected static $permitted_types = [
+            'master',
+            'slave',
+            'native',
+        ];
```

Comment:
Indention is off in this block.

Reply:
Fixed in following commit.
https://github.com/LithiumHosting/apnscp-powerdns/pull/4/commits/c59a47f1c497eeef3e3526d498e2a3e13a4eb1ab